install-dev/upgrade/php/create_multistore.php

```diff
@@ -44,7 +44,7 @@ function create_multistore()
     $all_themes_dir = _PS_ROOT_DIR_.DIRECTORY_SEPARATOR.'themes';
     $themes = scandir($all_themes_dir);
     foreach ($themes as $theme) {
-        if (!is_file($all_themes_dir.DIRECTORY_SEPARATOR.$theme) && is_dir($all_themes_dir.DIRECTORY_SEPARATOR.$theme.DIRECTORY_SEPARATOR) && $theme[0] != '.' && $theme != 'prestashop'  && $theme != 'default-bootstrap') {
+        if (!is_file($all_themes_dir.DIRECTORY_SEPARATOR.$theme) && is_dir($all_themes_dir.DIRECTORY_SEPARATOR.$theme.DIRECTORY_SEPARATOR) && $theme[0] != '.' && $theme != 'prestashop'  && $theme != _PS_DEFAULT_THEME_NAME_) {
```

Comment:
to be consistent with changes in `install-dev/controllers/http/system.php `, i think new line should look like this:

`... && $theme != 'prestashop'  && $theme != 'default-bootstrap'  && $theme != _THEME_NAME_`